### ISSUE TITLE
Add check if string that comes in (should not be array)

### ISSUE THREAD

**Issue by eklem** [eklem/words-n-numbers#188]
Fail a lot when writing tests for the stopword module when this is not in place. Just a "input is not a string"-message.

**Comment by eklem** [comment]
Use `typeOf` check: https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string-in-javascript/4059166#4059166


```javaScript
if (typeOf varName !== `string`) {
  throw 'Error: Input is not a string'
} 
```

**Comment by eklem** [comment]
Fixed in v7.1.0: https://github.com/eklem/words-n-numbers/releases/tag/v7.1.0
